.github/workflows/ci.yml

```diff
@@ -1,31 +1,41 @@
 name: ci
 on:
   push:
-   branches:
+    branches:
       - master
   pull_request:
 jobs:
-  build:
+  build-and-test:
```

Comment:
please keep this name unchanged, it's the same across all repos and it will require changes to the repo settings if you change the name